docs/public/configuration/profiles.md

```diff
@@ -5,8 +5,11 @@
 The default profile is documented in the [profile properties] section
 of the Zonemaster::Engine::Profile module.
 
-The default profile is stored in a default profile file, [profile.json],
-always loaded by Zonemaster-Engine.
+The default profile can be extracted from Zonemaster-Engine using this command.
+
+```sh
+perl -MZonemaster::Engine::Test -E 'say Zonemaster::Engine::Profile->default->to_json' | jq -S .
+```
```

Comment:
Here I think we should be more helpful.

````
The default profile can be extracted from Zonemaster-Engine to a file, e.g. `profile.json`, using this command.

```sh
perl -MZonemaster::Engine::Test -E 'say Zonemaster::Engine::Profile->default->to_json' | jq -S . > profile.json
```
````

Reply:
Updated.